lighthouse-logger/package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "lighthouse-logger",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "license": "Apache-2.0",
   "dependencies": {
     "debug": "^2.6.8",
     "marky": "^1.2.0"
+  },
+  "devDependencies": {
+    "@types/debug": "^0.0.31"
```

Comment:
none of these types are exposed in our API right?

Reply:
correct